packages/backend/src/accounting/tigerbeetle/utils.ts

```diff
@@ -60,6 +60,9 @@ export function tbTransferToLedgerTransfer(
     transferRef: fromTigerBeetleId(tbTransfer.user_data_128),
     type: transferTypeFromCode(tbTransfer.code),
     state: state,
-    ledger: tbTransfer.ledger
+    ledger: tbTransfer.ledger,
+    expiresAt: new Date(
+      Number(tbTransfer.timestamp / 1_000_000n) + tbTransfer.timeout
+    )
```

Comment:
```suggestion
      Number(tbTransfer.timestamp / 1_000_000n) + tbTransfer.timeout * 1000
```
(since it looks like [timeout](af79b50b-dd15-4684-b9a3-788832a47e0a) is in seconds)